core/dotnet2.2/CHANGELOG.md

```diff
@@ -23,3 +23,15 @@
 ## 1.13
 Changes:
 - Initial release
+
+## 1.14
+Changes:
+- Support for async methods. Example:
+
+```csharp
+        public async Task<JObject> MainAsync(JObject args)
+        {
+            await Task.Delay(10); // Just do a delay to have an async/await process occur.
+            return (args);
```

Comment:
shouldn't there be a Task.run somewhere for the awaitable method?

Reply:
no, no, Task.Run is only used in sync methods

Reply:
😅 that's how much `c#` i know :)